packages/extensions/src/index.ts

```diff
@@ -102,6 +102,16 @@ const normalizeExtension = (ext: ExtensionFormatLoose | any): ExtensionFormat =>
     return ext;
 };
 
+/**
+ * Get the priority for an extension.
+ * @ignore
+ * @param ext - Any extension
+ * @param defaultPriority - Fallback priority if none is defined.
+ * @returns The priority for the extension.
+ */
+const normalizePriority = (ext: ExtensionFormatLoose | any, defaultPriority: number): number =>
```

Comment:
Just an advice, for top level functions `normalizePriority` and `normalizeExtension`, I prefer `function xxx(...) { ... }` to `const xxx = (...) => { ... }`.